eng/helixcorefxtests.proj

```diff
@@ -101,6 +101,12 @@
     <TestHostRootPath>$(TestWorkingDir)testhost\</TestHostRootPath>
     <TestArchiveRuntimeRoot>$(TestWorkingDir)helix\</TestArchiveRuntimeRoot>
 
+    <!-- The VM PDB symbols for Windows live in the same directory as the VM binaries. Set a relative
+         path within TestHostRootPath (root of the correlation payload) that we can use to set the
+         symbol path for the purpose of getting a good stack trace for debug build VM asserts.
+    -->
+    <TestHostPDBPath>shared\$(MicrosoftNETCoreAppPackage)\$(ProductVersion)</TestHostPDBPath>
```

Comment:
Is there any way to make this a full path?

Reply:
It becomes a full path later, when combined with the dynamically determined `%HELIX_CORRELATION_PAYLOAD%`